delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/migration/MigrationV16ToV17.scala

```diff
@@ -0,0 +1,178 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.migration
+
+import akka.actor.typed.ActorSystem
+import akka.stream.alpakka.cassandra.scaladsl.CassandraSession
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.ElasticSearchViews
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.migration.MigrationV16ToV17._
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ElasticSearchViewEvent.{ElasticSearchViewCreated, ElasticSearchViewUpdated}
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ElasticSearchViewValue.IndexingElasticSearchViewValue
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.{ElasticSearchViewEvent, ElasticSearchViewType, ElasticSearchViewValue}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.JsonLdContext.keywords
+import ch.epfl.bluebrain.nexus.delta.sdk.migration.Migration
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.views.pipe._
+import ch.epfl.bluebrain.nexus.delta.sourcing.config.CassandraConfig
+import ch.epfl.bluebrain.nexus.delta.sourcing.utils.CassandraUtils
+import com.datastax.oss.driver.api.core.cql.PreparedStatement
+import com.typesafe.scalalogging.Logger
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.deriveConfiguredEncoder
+import io.circe.parser._
+import io.circe.syntax.EncoderOps
+import io.circe.{Encoder, Error, Printer}
+import monix.bio.Task
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+import java.util.UUID
+import scala.annotation.nowarn
+
+/**
+  * Migrate elasticsearch views to new model with pipeline
+  */
+final class MigrationV16ToV17 private (
+    session: CassandraSession,
+    config: CassandraConfig
+) extends Migration {
+
+  private val selectViewEvents =
+    s"SELECT persistence_id, event, sequence_nr, timestamp FROM ${config.keyspace}.tag_views where tag_name = ? limit 1000000 allow filtering;"
```

Comment:
As there are few events for the tag `elasticsearch`, the query with the allow filtering remains performant